src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/MediaSection.js

```diff
@@ -2,29 +2,49 @@
 import React from 'react';
 import type {ElementRef} from 'react';
 import {List, ListStore} from 'sulu-admin-bundle/containers';
+import {Button, ButtonGroup} from 'sulu-admin-bundle/components';
+import {translate} from 'sulu-admin-bundle/utils';
 
 type Props = {|
     adapters: Array<string>,
     listStore: ListStore,
     mediaListRef?: (?ElementRef<typeof List>) => void,
     onMediaClick: (mediaId: string | number) => void,
+    onUploadClick: (() => void) | null,
 |};
 
 export default class MediaSection extends React.PureComponent<Props> {
     handleMediaClick = (mediaId: string | number) => {
         this.props.onMediaClick(mediaId);
     };
 
+    handleUploadClick = () => {
+        if (this.props.onUploadClick) {
+            this.props.onUploadClick();
+        }
+    };
+
     render() {
         const {
             adapters,
             listStore,
             mediaListRef,
+            onUploadClick,
         } = this.props;
 
         return (
             <List
                 adapters={adapters}
+                buttons={[
```

Comment:
would forward a `listActions` prop instead of abstracting this behind a `onUploadClick` prop.

IMO we should remove this component in the future as it does not really do anything. but we need to think about backwards compatibility when doing that. so i would leave it for now and not add any logic to it to keep things simple 🙂